src/containers/Roadmap/components/TeamRoadmapCard/TeamRoadmapCard.scss

```diff
@@ -42,7 +42,11 @@
 
   &__content {
     max-height: 480px;
-    overflow-y: scroll;
+    overflow-y: auto;
+
+    &::-webkit-scrollbar-track {
+      display: none;
+    }
```

Comment:
@jamessspanggg I think it should work without this too. You can try on mac first, I am leaving this here.

We can remove this in some other PR, if all works fine on your end.